packages/smart-print/smart-print.0.3.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+
+maintainer: "dev@clarus.me"
+authors: ["Guillaume Claret <dev@clarus.me>"]
+homepage: "https://github.com/clarus/smart-print"
+bug-reports: "https://github.com/clarus/smart-print/issues"
+license: "BSD-3-Clause"
+
+depends: [
+  "ocaml" {>= "4.00.0"}
+  "dune" {>= "2.0"}
+  "odoc" {with-doc}
+]
```

Comment:
Could you add a dependency on `ocamlfind`? Calling `dune install` requires `ocamlfind` (see https://github.com/ocaml/dune/issues/3025)